src/app/view/components/dot-toolbar/dot-toolbar.component.ts

```diff
@@ -33,7 +33,9 @@ export class ToolbarComponent implements OnInit {
 
     siteChange(site: Site): void {
         this.siteService.switchSite(site);
-        this.dotNavigationService.goToFirstPortlet();
+        if (this.route.snapshot['_routerState'].url.indexOf('edit-page/') >= 0) {
```

Comment:
I think we should create a method in the `dotRouterService` like `isEditPage()` to do this. @freddyucv @hmoreras?

Reply:
sounds like it will be reusable. agree.